server/http_handler_test.go

```diff
@@ -221,7 +221,8 @@ func (ts *HTTPHandlerTestSuite) TestRegionsFromMeta(c *C) {
 }
 
 func (ts *HTTPHandlerTestSuite) startServer(c *C) {
-	mvccStore := mocktikv.NewMvccStore()
+	mvccStore, err := mocktikv.NewMVCCLevelDB("")
```

Comment:
DRY, could you create a a function like MustNewStore and check err in the function?